src/pet_mad/_models.py

```diff
@@ -5,11 +5,21 @@
 
 from metatomic.torch import AtomisticModel
 from metatrain.utils.io import load_model as load_metatrain_model
-from .utils import get_metadata
+from metatrain.utils.io import _hf_hub_download_url
```

Comment:
please don't import private functions from metatrain, these can be changed at any time which would break this package